README.md

```diff
@@ -41,23 +48,38 @@ resolved Promise.
 
 ### Generate a DID Document
 
-Generate a new DID document.
+#### Generating and Registering a Veres One DID Document
```

Comment:
The registration part is duplicated in next section.  Can this section be just the generation part?

Reply:
Fixed, thanks.